File: Scrapers/hifipunkten.py
from Scrapers.common import AshopCategoryScraper


class HiFiPunktenScraper(AshopCategoryScraper):
    def __init__(self):
        super().__init__(
            base_url="https://www.hifi-punkten.se",
            name="HiFi-Punkten",
            category_url="https://www.hifi-punkten.se/kategori/1/produkter",
        )

File: ts-scrapers/src/sites/hifipunkten.ts
import { AshopCategoryScraper } from '../bases/ashop.js';

export class HiFiPunktenScraper extends AshopCategoryScraper {
  constructor() {
    super({
      name: 'HiFi-Punkten',
      baseUrl: 'https://www.hifi-punkten.se',
      categoryUrl: 'https://www.hifi-punkten.se/kategori/1/produkter',
      slug: 'hifi-punkten',
    });
  }
}
